docpages/example_code/using_emoji.cpp

```diff
@@ -0,0 +1,46 @@
+#include <dpp/dpp.h>
+#include <dpp/unicode_emoji.h>
+
+int main() {
+    dpp::cluster bot("Epic Token", dpp::i_default_intents | dpp::i_message_content);
+    /* The second argument is a bitmask of intents - i_message_content is needed to see the messages */
+
+    bot.on_log(dpp::utility::cout_logger());
+
+    /* We'll be using a shocked guy emoji */
+    dpp::emoji shocked("vahuyi", 1179366531856093214);
+
+    bot.on_message_create([&bot, shocked](const dpp::message_create_t& event) {
+        if (event.msg.content == "WHAT?") {
+            /* If some unknown content shocks the user */
+
+            bot.message_add_reaction(event.msg.id, event.msg.channel_id, shocked.format());
+            /* React to their message with a shocked guy */
+        } else if (event.msg.content == "I'm hungry") {
+            /* But if they're hungry */
+
+            bot.message_add_reaction(event.msg.id, event.msg.channel_id, dpp::unicode_emoji::cut_of_meat);
+            /* Let's send some meat to the message, so they don't starve. They will thank us later. */
```

Comment:
I would put this above the `bot.message_add_reaction`.

Reply:
But there's a comment below the `if` and I can't put it above the statement

Reply:
that's fine, it makes sense to bunch them together. You could even put them in the same comment and that still makes sense!